org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/highlighting/SemanticHighlightings.java

```diff
@@ -0,0 +1,1021 @@
+/*******************************************************************************
```

Comment:
if file was copied from jdt, then it should retain the original copyright from IBM.

Add Contributors section:
```xml
 *
 * Contributors:
 *     IBM Corporation - initial API and implementation
 *     TypeFox - port to jdt.ls
```